pkgs/development/tools/run/default.nix

```
@@ -0,0 +1,22 @@
+{ stdenv, buildGoModule, fetchFromGitHub }:
+buildGoModule rec {
+  pname = "run";
+  version = "0.7.0";
+
+  src = fetchFromGitHub {
+    owner = "TekWizely";
+    repo = "run";
+    rev = "v${version}";
+    sha256 = "04p112ciyn1zc86is49rjdvv0a0vbiyhqbhqr468f4iz3ifmfis5";
```

Having a hash mismatch (double-checked with `nix-prefetch-github --rev v0.7.0 TekWizely run`)

```
hash mismatch in fixed-output derivation '/nix/store/yxv3d0s2ckazjy21xh3m0xla3hi11ind-source':
  wanted: sha256:04p112ciyn1zc86is49rjdvv0a0vbiyhqbhqr468f4iz3ifmfis5
  got:    sha256:0365nvsqrlagrp08sifbdk3rgy7r4hmp3sx5zhizamadfcj2fsv6
```

```suggestion
    sha256 = "0365nvsqrlagrp08sifbdk3rgy7r4hmp3sx5zhizamadfcj2fsv6";
```